dxr/testing.py

```diff
@@ -305,6 +319,16 @@ def _make_file(path, filename, contents):
         file.write(contents.encode('utf-8'))
 
 
+def _format_dict_values(dictionary, *args, **kwargs):
```

Comment:
I like that a lot better, I wasn't quite happy with what I had after the rebase but didn't give it enough thought before pushing. :D